client/src/components/History/CurrentHistory/HistoryDetails.vue

```diff
@@ -36,6 +40,14 @@ export default {
         history: { type: Object, required: true },
         writeable: { type: Boolean, default: true },
     },
+    computed: {
+        shownCount() {
+            return store.getters.getHistoryItems({ historyId: this.history.id, filterText: "" }).length;
```

Comment:
It would be nice to put these getters directly into the history store as `historyCount` and `historyHiddenCount` or something for reuse and caching